source/rrStringUtils.cpp

```diff
@@ -638,6 +639,19 @@ double toDouble(const string& str)
     return std::numeric_limits<double>::quiet_NaN();
 }
 
+vector<string> toStringVector(const string& str)
+{
+	// string vector is in the form "[s1,s2,s3...sn]"
+	// get rid of '[' and ']' first
+
+	size_t start = str.find("[");
+	string res = str.substr(start + 1, str.size() - 1);
+	// parse into string vector
+	vector<string> parts(splitString(res, ","));
```

Comment:
Are the strings in the string vector surrounded by quotes? Also, are there allowed to be whitespace characters between the commas and the strings and betwen the strings and the brackets?